signer/src/network/mod.rs

```diff
@@ -7,6 +7,9 @@
 #[cfg(any(test, feature = "testing"))]
 pub mod in_memory;
 
+#[cfg(any(test, feature = "testing"))]
+pub mod in_memory2;
```

Comment:
I figure we can clean this up in a separate PR which refactors the rest of the tests